f8a_worker/workers/graph_importer.py

```diff
@@ -35,7 +35,8 @@ def execute(self, arguments):
             {
                         'ecosystem': arguments['ecosystem'],
```

Comment:
`arguments['ecosystem']` could be for example `redhat-maven`. You need to do something like https://github.com/msrb/fabric8-analytics-worker/blob/8f069e308df89ebdab8c89f0e63adea1cd52294d/f8a_worker/workers/CVEchecker.py#L218-L219 to translate ecosystem string to Ecosystem object and from there you should be able to find corresponding ecosystem backend ('maven' for ecosystem 'redhat-maven').

Reply:
@msrb, How would we determine what is the backend then?